gvr-vuforia/app/src/main/java/org/gearvrf/vuforiasample/VuforiaSampleMain.java

```diff
@@ -169,7 +187,12 @@ private void hideTeapot() {
         }
     }
 
-    public void updateObjectPose(State state) {
+    public void updateObjectPose() {
+        State state = mRenderer.begin();
+
+        // viewport as per default 1024 x 1024 window size
+        mainScene.getMainCameraRig().setViewport(0, 224, 1024, 576);
```

Comment:
The camera resolution is 2560 X 1440 giving aspect ratio as 16:9. The GL viewport aspect ratio also match. So just change 1024 X 1024 to 16:9